modcc/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# The modcc compiler is split into a staric library and driver;
+# The modcc compiler is split into a static library and driver;
 # unit tests for the driver also use this library.
 
 set(libmodcc_sources
```

Comment:
Question: If given an external modcc, should we even attempt to build an internal modcc? _Do_ we do that currently? Similar: tests!

Reply:
I assume the internal lib is needed in any case, is that correct? I suppose the whole `modcc` block can go in the if clause, I'll give it a go.

Reply:
It goes.

Tests: it's a bit hard to build two version of modcc from one copy of the codebase. My initial idea was to make that part of the build of `a-o-l`.